infra/examples-dev/gcp/main.tf

```diff
@@ -27,9 +27,50 @@ provider "google" {
   impersonate_service_account = var.gcp_terraform_sa_account_email
 }
 
+locals {
+  host_platform_id = "GCP"
+}
+
+# TODO: this has 5 remote modules; combine some?
+#  eg, worklytics-connectors + gcp-host + worklytics-psoxy-connection-generic into a single
+#     gcp-host-for-worklytics? poor TF style, but simplifies root module?
+
+# in effect, these are for sources for which authentication/authorization cannot (or need not)
+# be provisioned via Terraform, so doesn't add any dependencies
+# call this 'generic_source_connectors'?
+module "worklytics_connectors" {
+  source = "../../modules/worklytics-connectors"
+
+  enabled_connectors    = var.enabled_connectors
+  example_jira_issue_id = var.example_jira_issue_id
+  jira_cloud_id         = var.jira_cloud_id
+  jira_server_url       = var.jira_server_url
+  salesforce_domain     = var.salesforce_domain
```

Comment:
we should probably combine this under an object `connector_settings`, so it can grow just changing the description of the variable rather than passing every single one here.

Reply:
I don't agree.

It would make configuration in the `terraform.tfvars` file less straight-forward. To add sources, you couldn't just append to it; but would have to find the `connector_settings` variable within that and modify it to add what you want.